libs/wingsdk/src/core/awscdk-app.ts

```diff
@@ -0,0 +1,81 @@
+import { Construct } from "constructs";
```

Comment:
I would define this all in `target-awscdk/app.ts` since the logic does not need to be shared with any other targets

Reply:
Great 😃

It's really make more sense